From: Zijun Hu <quic_zijuhu@quicinc.com>
Subject: cpu: remove needless return in void API suspend_enable_secondary_cpus()
Date: Fri, 21 Feb 2025 05:02:07 -0800

Remove needless 'return' in void API suspend_enable_secondary_cpus() since
both the API and thaw_secondary_cpus() are void functions.

Link: https://lkml.kernel.org/r/20250221-rmv_return-v1-2-cc8dff275827@quicinc.com
Signed-off-by: Zijun Hu <quic_zijuhu@quicinc.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/cpu.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/cpu.h~cpu-remove-needless-return-in-void-api-suspend_enable_secondary_cpus
+++ a/include/linux/cpu.h
@@ -148,7 +148,7 @@ static inline int suspend_disable_second
 }
 static inline void suspend_enable_secondary_cpus(void)
 {
-	return thaw_secondary_cpus();
+	thaw_secondary_cpus();
 }
 
 #else /* !CONFIG_PM_SLEEP_SMP */
_
